wallet-test/src/test/scala/org/bitcoins/wallet/BitcoindBlockPollingTest.scala

```diff
@@ -0,0 +1,77 @@
+package org.bitcoins.wallet
+
+import org.bitcoins.core.currency._
+import org.bitcoins.db.AppConfig
+import org.bitcoins.rpc.util.AsyncUtil
+import org.bitcoins.server.{BitcoinSAppConfig, BitcoindRpcBackendUtil}
+import org.bitcoins.testkit.fixtures.BitcoinSFixture
+import org.bitcoins.testkit.util.BitcoinSAsyncTest
+import org.bitcoins.testkit.wallet.BitcoinSWalletTest
+import org.bitcoins.testkit.{BitcoinSTestAppConfig, EmbeddedPg}
+import org.bitcoins.wallet.config.WalletAppConfig
+
+import scala.concurrent.Await
+import scala.concurrent.duration.DurationInt
+
+class BitcoindBlockPollingTest extends BitcoinSAsyncTest with EmbeddedPg {
```

Comment:
Any reason you don't just use the `BitcoinSWalletTest` test trait? I believe that has everything you need?

Reply:
Because the wallet's in there are automatically setup to process new blocks. We want to test that our polling will find the new blocks